### PR TITLE
[Improvement] Move detailed client configuration to individual doc

### PR DESCRIPTION
**Pull request by frankliee**
### What changes were proposed in this pull request?
1.  Put detailed configuration to doc subdirectory.
2. Add doc for client quorum setting.

### Why are the changes needed?
Update doc

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
Just doc.

### CODE REVIEW

**Comment by colinmjj** [comment]
@frankliee Add a section to describe how to use this feature with config?

**Comment by frankliee** [comment]
> @frankliee Add a section to describe how to use this feature with config?

OK

**Comment by jerryshao** [comment]
I would suggest not add everything to readme, instead trying to use docs for the details. please think about it. CC @colinmjj

**Comment by colinmjj** [comment]
> I would suggest not add everything to readme, instead trying to use docs for the details. please think about it. CC @colinmjj

Yes, there should be docs for new comer to know & use Firestorm better.

**Comment by frankliee** [comment]
> I would suggest not add everything to readme, instead trying to use docs for the details. please think about it. CC @colinmjj

I will push a new PR that contains TOC and separated docs about client/server/coordinator.

File: README.md [hunk 1]
```diff
@@ -217,27 +217,9 @@ The important configuration is listed as following.
 |rss.server.flush.cold.storage.threshold.size|64M| The threshold of data size for LOACALFILE and HDFS if MEMORY_LOCALFILE_HDFS is used|
 
 
-### Spark Client
+### Shuffle Client
```
Comment (jerqi) [inline README.md:220]: Why is Shuffle Client?

File: docs/coordinator_guide.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+displayTitle: Firestorm Coordinator Guide
+title: Firestorm Coordinator Guide
+description: Firestorm Coordinator Guide
+---
+
+# Firestorm Coordinator Guide
```
Comment (jerqi) [inline docs/coordinator_guide.md:8]: We can some description about `dynamicEnable` and how to use DelegateShuffleManager.

Comment (frankliee) [reply docs/coordinator_guide.md:8]: There will be two extra PRs for coor and server

File: README.md (hunk 1 above)
Comment (frankliee) [reply README.md:220]: The linked page contains both Spark and MR clients

**Comment by jerqi** [comment]
LGTM.

**Review by colinmjj — APPROVED** [review]
LGTM

**Pull request merged**